package.json

```
@@ -27,10 +27,9 @@
     "start": "react-app-rewired start",
     "build": "react-app-rewired build",
     "test": "react-app-rewired test",
-    "lint": "npm run lint:editorconfig && npm run lint:eslint",
-    "lint:editorconfig": "editorconfig-checker",
-    "lint:eslint": "eslint --ext .js,.jsx,.ts,.tsx src",
-    "lint:eslint:fix": "npm run lint:eslint -- --fix",
+    "lint": "eslint src --ext .ts,.tsx",
+    "lint:fix": "eslint src --ext .js,.jsx,.ts,.tsx --fix",
+    "format": "prettier --write \"src/**/*.{js,jsx,ts,tsx,json}\"",
```

```suggestion
    "lint:eslint": "eslint src --ext .ts,.tsx",
    "lint:prettier": "prettier --check \"src/**/*.{js,jsx,ts,tsx,json}\"",
    "format:eslint": "eslint src --ext .js,.jsx,.ts,.tsx --fix",
    "format:prettier": "prettier --write \"src/**/*.{js,jsx,ts,tsx,json}\"",
```
We want our lint/format scripts to be able to run both (might be worth adding npm-run-all to expose lint/format to run both)